README.md

```diff
@@ -146,9 +146,9 @@ Or
 yarn cypress:open
 ```
 
-It will open cypress dashboard, through which you need to select `E2E Testing`.
-Once you done with selection you will get options to choose your preferred browser in which you want to run the test.!
-Once you done with selecting the browser you need to click on `Start E2E Testing in <SELECTED_BROWSER_NAME>.` Selected browser will popup and you can able to see list of cypress test, click on the test to start testing.!
+It will open the cypress dashboard, through which you need to select `E2E Testing`.
+Once you are done with the selection you will get options to choose your preferred browser in which you want to run the test.!
+Once you are done with selecting the browser you need to click on `Start E2E Testing in <SELECTED_BROWSER_NAME>.` Selected browser will popup and you can able to see a list of cypress tests, click on the test to start testing.!
```

Comment:
```suggestion
It will open the cypress dashboard, through which you need to select `E2E Testing`.
Once you are done with the selection you will get options to choose your preferred browser in which you want to run the test.!
Once you select the browser you need to click on `Start E2E Testing in <SELECTED_BROWSER_NAME>.` The chosen browser will pop up and you can see a list of cypress tests, click on the test to start testing.!
```

Reply:
@KashishLakhara04 @Sachin-chaurasiya 

Can you please confirm if this comment has been addressed? I am about to merge it.

Reply:
I see its incorporated

Reply:
Yes we can merge it.